adapters/Pangle/Public/Headers/GADMediationAdapterPangle.h

```diff
@@ -0,0 +1,49 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import <GoogleMobileAds/GoogleMobileAds.h>
+
+typedef NS_ENUM(NSInteger, GADPangleErrorCode) {
+    /// Missing server parameters.
+    GADPangleErrorInvalidServerParameters = 101,
+    /// Banner size mismatch.
+    GADPangleErrorBannerSizeMismatch = 102,
+};
+
+@interface GADMediationAdapterPangle : NSObject <GADRTBAdapter>
+
+/// Set the COPPA setting before initialize in Pangle SDK.
+///
+/// @param COPPA An integer value that indicates whether the app should be treated as
+/// child-directed for purposes of the COPPA.  0 means false. 1 means true. -1 means
+/// unspecified. Any value outside of -1, 0, or 1 will result in this method being a no-op.
++ (void)setCOPPABeforeInitialize:(NSInteger)COPPA;
```

Comment:
Renaming won't resolve the issue.

This (i.e. `+ (void)setCOPPABeforeInitialize:(NSInteger)COPPA;`) should be removed in this header file because we shouldn't expose this API to the publishers.